config-model/src/main/java/com/yahoo/config/model/deploy/DeployState.java

```diff
@@ -263,6 +266,22 @@ public boolean isHosted() {
 
     public Optional<TlsSecrets> tlsSecrets() { return properties.tlsSecrets(); }
 
+    public Optional<String> tlsClientAuthority() {
+        var caFile = applicationPackage.getClientSecurityFile();
+        if (caFile.exists()) {
+            try {
+                var caPem = IOUtils.readAll(caFile.createReader());
+                return Optional.of(caPem);
+            } catch (FileNotFoundException e) {
```

Comment:
It probably can't since we check for existing file earlier and the application package is immutable - but it's one of the declared checked exceptions on the method so I thought I might as well handle it.